SUMMARY.md

```diff
@@ -19,6 +19,7 @@
 * App Developers
   * [Receive tokens](guides/app-developers/receive-tokens.md)
   * [Bridge tokens](guides/app-developers/bridge-tokens.md)
+  * [EthStorage App Guide](guides/app-developers/es-app-guide.md)
```

Comment:
EthStorage App Guide  => EthStorage Usage Guide ?

Reply:
Done in 474b84f52ea1c2b8b2a5c2856a21df2f8fa1143e.